packages/protocol-kit/scripts/safe-deployments/updateLocalNetworks.ts

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import { getChainShortName, getLocalNetworksConfig, getSafeDeploymentNetworks } from './utils'
+import { networks } from '../../src/utils/eip-3770/config'
+
+interface NetworkShortName {
+  shortName: string
+  chainId: bigint
+}
+
+/**
+ * Updates the local networks array in the configuration file.
+ *
+ * @param {NetworkShortName[]} networks - The full list of networks.
+ */
+
+function updateLocalNetworks(networks: NetworkShortName[]) {
+  const path = 'src/utils/eip-3770/config.ts'
+
+  fs.readFile(path, (err, data) => {
+    if (err) {
+      console.error(err)
+      return
+    }
+
+    const content = data.toString()
+    const startIndex =
+      content.indexOf('export const networks: NetworkShortName[] = [') +
+      'export const networks: NetworkShortName[] = ['.length
+    const endIndex = content.indexOf(']\n\nif (process.env.TEST_NETWORK ===')
+
+    const sortedNetworks = networks
+      .sort((a, b) => Number(a.chainId - b.chainId))
+      .map(
+        (network, index) =>
+          `  { chainId: ${network.chainId}n, shortName: '${network.shortName}' }${index === networks.length - 1 ? '' : ','}`
```

Comment:
The problem with this thing is that, because we are assembling bits of content in different statements, its a hard to understand the outgoing structure. Life would be much kinder to us if it looked more like "stuffing" a template:
```
`
    export const networks: NetworkShortName[] = [
        ${sortedNetworks.map(network => `
        {
            chainId: '${network.name}',
            shortName: '${network.shortName}'
        }`).join(',')}
    ]
`
```